pkg/networkpath/traceroute/runner.go

```diff
@@ -173,7 +173,7 @@ func getPorts(configDestPort uint16) (uint16, uint16, bool) {
 	return destPort, srcPort, useSourcePort
 }
 
-func (r *Runner) processResults(res *results.Results, hname string, destinationHost string, destinationIP net.IP) (payload.NetworkPath, error) {
+func (r *Runner) processResults(res *results.Results, hname string, destinationHost string, destinationPort uint16, destinationIP net.IP) (payload.NetworkPath, error) {
```

Comment:
🥜 : on the surface I feel like a signature of 

```go
processResults(res *results.Results, hname string, destinationHost string, destinationIP net.IP, destinationPort uint16)
```

is better because then it flows more naturally to how it would be written `ip:port`. At the end of the day either will work so I won't block for this but just a thought